tests/unit/components/settings/settings.spec.js

```diff
@@ -26,10 +30,12 @@ describe('NavBar', () => {
 
   beforeEach(() => {
     state = {
-      pnp: {
-        peerConnectionStatus: jest.fn(),
-        edgePeerId: jest.fn()
-      }
+      pnp: cloneDeep(pnpStoreModule),
+      // pnp: {
```

Comment:
@vickywane do we need to commit code that's commented out. Can you please clean up.

Reply:
I have removed the comment/